docs/04_web/05_micro-front-ends/04_foundation-login/04_foundation-login.mdx

```diff
@@ -61,7 +61,7 @@ To enable this micro front-end in your application, follow the steps below.
 This page assumes you're using the routing systems that are part of `foundation-ui`, and will cover set-up as part of that routing system.
 :::
 
-Once installed, you can either use `Login` as is or re-configure it. As with all our micro front-ends, you also have the option of lazy loading it.
+Once installed, you can either use `Login` as is or re-configure it. As with all our micro front-ends, you also have the option of lazy-loading it.
```

Comment:
question:

not sure why GH is highlighting a few words here with a "dark red" style..